internal/pkg/cli/init.go

```diff
@@ -145,15 +141,17 @@ func newInitOpts(vars initVars) (*initOpts, error) {
 			name:         defaultEnvironmentName,
 			isProduction: false,
 		},
-		store:            configStore,
-		appDeployer:      deployer,
-		prog:             spin,
-		prompt:           prompt,
-		identity:         id,
-		appVersionGetter: appVersionGetter,
-		appCFN:           cloudformation.New(defaultSess, cloudformation.WithProgressTracker(os.Stderr)),
-		sess:             defaultSess,
-		templateVersion:  version.LatestTemplateVersion(),
+		store:       configStore,
```

Comment:
i can't comment on the lines i want😭 but i wonder if we should avoid setting `initEnvVars.appName` in general here, as it _only_ works if flags were passed in?

Reply:
ok done!